CMakeLists.txt

```diff
@@ -198,6 +187,14 @@ if(NOT PERFSTUBS_LIBRARY_ONLY)
     if (PERFSTUBS_USE_DEFAULT_IMPLEMENTATION)
         add_library(implementation implementation/tool1_implementation.cpp)
         add_library(implementation2 implementation/tool2_implementation.cpp)
+        target_include_directories(implementation PRIVATE
+          $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+          $<INSTALL_INTERFACE:include>
+        )
+        target_include_directories(implementation2 PRIVATE
+          $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+          $<INSTALL_INTERFACE:include>
+        )
```

Comment:
I think there may be a cleaner way of doing this, that would be useful for other tools implementations. My understanding is that tools shouldn't and can't link to perfstubs; they are the impl not the stub. However they do all have common include needs - so an interface only library could be defined with the necessary includes.